src/android/com/adobe/phonegap/push/GCMIntentService.java

```diff
@@ -119,6 +122,49 @@ private void replaceKey(String oldKey, String newKey, Bundle extras, Bundle newE
     }
 
     /*
+     * Normalize localization for key
+     */
+    private String localizeKey(Context context, String key, String value) {
+        if (key.equals(TITLE) || key.equals(MESSAGE) || key.equals(SUMMARY_TEXT)) {
+            try {
+                JSONObject localeObject = new JSONObject(value);
```

Comment:
I agree with my earlier comment and @fredgalvao. I said `something like` because I'd like to get rid of the JSONExceptions.